docs/src/apireference.md

```diff
@@ -404,6 +404,24 @@ ModifyObjectiveNotAllowed
 DeleteNotAllowed
 ```
 
+## Models
+
+The interface is generic and does not have a fixed list of functions and sets.
```

Comment:
Suggested text:

MOI is designed to be extensible, so there is no fixed list of possible functions and sets. This makes it challenging to define efficient storage representations for MOI models. For cases where the functions and sets of interest are known in advance (for example, solvers support a fixed list of functions and sets), we provide the [`Utilities.@model`](@ref) that macro defines a [`ModelLike`](@ref) given a list of functions and sets to support.

[`Utilities.UniversalFallback`](@ref) is a layer that sits on top of any `ModelLike` and provides non-specialized (slower) fallbacks for constraints and attributes that the underlying `ModeLike` does not support.

Reply:
That's better :)